tools/tfuser/main.go

```diff
@@ -27,6 +29,18 @@ func requireSeed(c *cli.Context) error {
 	return nil
 }
 
+type userIdentity struct {
```

Comment:
This seems to be exactly the same struct as used in tffarmer, is there no way to only declare this once in a common file and then import it in both places?

Reply:
it is indeed exactly the same, but I'm not sure where we could put it.

Reply:
Given that this identity is used by multiple tools packages, I'd say it could just be moved to a new `identity` pkg in the tools repo?